constants/constants.go

```diff
@@ -9,11 +9,11 @@ const OpLogRequestValue = "TECHRAIL_GROUND_OPLOG_REQUEST_VALUE"
 
 const (
 	RedisDefaultPassword                    = ""
-	RedisDefaultAddr                        = "localhost:6379"
+	RedisDefaultAddr                        = "localhost:7006"
 	RedisDefaultDbNumber                    = 0
 	RedisDefaultProtocol                    = 3
 	RedisDefaultOperationMode               = "auto"
-	RedisDefaultUrl                         = "redis://127.0.0.1:6379"
+	RedisDefaultUrl                         = "redis://127.0.0.1:7006"
```

Comment:
And why is the default port number changed here? Also, aren't we declaring this twice because we already did that in the cache package?